ga/server/periodic/ga-server-periodic.cpp

```diff
@@ -28,8 +41,11 @@
 #include "rtspconf.h"
 #include "controller.h"
 #include "encoder-common.h"
+#include "../../client/rttserver.h"
 
-#define	TEST_RECONFIGURE
+// #define	TEST_RECONFIGURE
+#define PKTBUF 512
+#define PKTPORT 8556
```

Comment:
If these are just going to be a hard define, we should put the client and server port in the same place. Like ../../client/rttserver.h which is already being included for some reason.